package/remux/remux.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=App launcher that supports multi-tasking applications
+After=xochitl.service opt.mount
+StartLimitInterval=30
+StartLimitBurst=5
+Conflicts=draft.service tarnish.service xochitl.service
```

Comment:
can it conflict with `xochitl.service` and also use `after xochitl.service` - what's that mean?

Reply:
This looks like an error indeed. Remux is not supposed to conflict with xochitl. Systemd seems to handle this correctly however, so this is not blocking. I’ll open a separate issue. Thanks!